alex/applications/PublicTransportInfoEN/data/expand_boroughs_script.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
```

Comment:
Well I admit that English expansions are not as overloaded, but it serves the purpose. And the purpose is to capture other forms (dialect, abbreviations etc), Sure, I can put them directly into database, It is just five boroughs and there are no other forms of any of those (YET). but I thought it was a good idea to maintain the same manner for the easy management of all places to be at one place only.